js/pages/tournamentlobbypage.ts

```diff
@@ -228,6 +235,40 @@ export class TournamentLobbyPage extends PageBase {
             });
             return sortedPrizes;
         }, this);
+        this.getPrizeStructureInformation = ko.computed(function () {
```

Comment:
Name of the property should not include `Get` or `Set` words. It better to be called `prizeStructureInformaiton`